test/utils.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/* eslint-env mocha */
+
+function getTimedPromise(fn, err, time) {
```

Comment:
Is this something we really have to invent ? Isn't there a standard way to do this ?

Reply:
I was wondering the same 😄

Reply:
Looks like it's not that far-fetched: https://stackoverflow.com/questions/22707475/how-to-make-a-promise-from-settimeout